apps/twitter/config/config.exs

```diff
@@ -1,30 +1,14 @@
-# This file is responsible for configuring your application
-# and its dependencies with the aid of the Mix.Config module.
 use Mix.Config
 
-# This configuration is loaded before any dependency and is restricted
-# to this project. If another project depends on this project, this
-# file won't be loaded nor affect the parent project. For this reason,
-# if you want to provide default values for your application for
-# 3rd-party users, it should be done in your "mix.exs" file.
+# Configure ExTwitter
```

Comment:
Are these comments necessary? It seems that `config :extwitter` is self-documenting.

Reply:
I usually follow the _Phoenix_ approach which adds a single line comment before each block...